test/RewardsStreamerMP.t.sol

```diff
@@ -1922,7 +2041,7 @@ contract LockTest is RewardsStreamerMPTest {
 
         _stake(alice, 10e18, 0);
 
-        vm.expectRevert(StakeMath.StakeMath__InvalidLockingPeriod.selector);
+        vm.expectRevert();
```

Comment:
This will be fixed in a future PR but the reason this has been remove is because, if we keep it, then the fuzzer finds cases where it reverts with `StakeManager__InvalidPeriod`.

If we expect the `StakeManager__InvalidPeriod` error, then it will find cases of invalid lock period by StakeMath.

The reason this happens is likely because it will find parameters where the `totalLockTime` does not exceed, but the lockPeriod provided is still considered invalid inside stake path because it still checks for `MIN_LOCKUP_TIME` 

We actually need to allow for lockPeriods that are less than `MIN_LOCKUP_TIME` if the vault has already been locked before and the new lock increase doesn't already exceeds `MIN_LOCKUPTIME` with th existing lock.